fn.go

```diff
@@ -148,10 +255,8 @@ func (f *Function) RunFunction(_ context.Context, req *fnv1beta1.RunFunctionRequ
 
 		fmt.Println("DueOrderTime is passed due, send results in slack chanel")
 	} else {
-		desired, err := request.GetDesiredComposedResources(req)
-		if err != nil {
-			response.Fatal(rsp, errors.Wrapf(err, "cannot get desired resources from %T", req))
-			return rsp, nil
+		if timeElapsed(xr, rsp) {
+			sendSlackMessage(xr)
 		}
 		pollServicePort, _ := strconv.Atoi(os.Getenv("POLL_SERVICE_PORT"))
 		pollServiceTargetPort, _ := strconv.Atoi(os.Getenv("POLL_SERVICE_TARGET_PORT"))
```

Comment:
@ca7alindev Please use function input

Reply:
@evghen7 Also for function input i have issue related for all input parameters #9